histogram.js

```diff
@@ -49,19 +49,11 @@ class HistogramElement extends HTMLElement {
     super();
 
     this.attachShadow({ mode: 'open' });
-
-    upgradeProperty(this, 'label');
+    
```

Comment:
Isn't this label a different thing? I thought this was for the chart titles, which we still need.

Reply:
Yes, they were used before in histogram and percent_box. Now we have already set up the chart title in the `<iobio-label-info-button>`. 

E.g: https://github.com/iobio/iobio-charts/blob/d6672a72a7aac960ba8060179274b40c9b5a591e/index.html#L148

Reply:
Ohh that's right! Good catch.